api/src/bigquerytest/java/org/pmiops/workbench/api/CohortBuilderControllerBQTest.java

```diff
@@ -33,7 +33,7 @@
 import org.pmiops.workbench.cohortbuilder.CohortBuilderServiceImpl;
 import org.pmiops.workbench.cohortbuilder.CohortQueryBuilder;
 import org.pmiops.workbench.cohortbuilder.SearchGroupItemQueryBuilder;
-import org.pmiops.workbench.cohortbuilder.mappers.CohortBuilderMapperImpl;
+import org.pmiops.workbench.cohortbuilder.mapper.CohortBuilderMapperImpl;
```

Comment:
Question: I know this isnt part of this PR but why do we need Impl class in Test why not just the interface?

Reply:
Because the Impl class needs to be injected to the real service, since this is an end to end integration test